static/app/components/events/interfaces/threads/threadSelector/selectedOption.tsx

```diff
@@ -18,7 +18,7 @@ type ThreadInfo = {
 
 function getThreadLabel(details: ThreadInfo) {
   const threadLabel = details?.label || `<${t('unknown')}>`;
-  return details.state ? `${threadLabel} (${details.state})` : threadLabel;
+  return details.state ? `${threadLabel}` : threadLabel;
```

Comment:
We could just change the whole function body to

```js
return details?.label || `<${t('unknown')}>`;
```

Reply:
lolol idk what i was trying to do here